cardano-api/src/Cardano/Api/LedgerState.hs

```diff
@@ -0,0 +1,849 @@
+{-# LANGUAGE DataKinds #-}
+{-# LANGUAGE DerivingStrategies #-}
+{-# LANGUAGE GADTs #-}
+{-# LANGUAGE GeneralizedNewtypeDeriving #-}
+{-# LANGUAGE LambdaCase #-}
+{-# LANGUAGE PatternSynonyms #-}
+{-# LANGUAGE RankNTypes #-}
+{-# LANGUAGE ScopedTypeVariables #-}
+{-# OPTIONS_GHC -freduction-depth=0 #-}
```

Comment:
I can seem to find any docs on this. Do you have a source?

Reply:
Wow.. not sure how that got there. I've removed it.